src/edu/ucsd/library/xdre/tab/FilesChecker.java

```diff
@@ -154,6 +172,10 @@ public boolean filesValidate ()
 			// Build the command for file validation
 			List<String> cmd = new ArrayList<>();
 			cmd.add( command );
+			if (src.length() > MAX_FILESIZE) {
```

Comment:
@escowles I found that the ExifTool only need large files support for files with gigabytes. I am wondering why it need us to enable it but not handle it automatically instead. Will it need pre-allocate some resources or check for system resources to handle the large files support? These could be an additional overhead for small files but I don't see any documentations for it. So I think it's fare for us to just enable it when necessary.